server/lib/models/account.go

```diff
@@ -77,6 +71,19 @@ func (a *Account) StringId() string {
 	return strconv.FormatUint(uint64(a.Id), ACCOUNT_ID_BASE)
 }
 
+func (a *Account) Persona(dbif Dbif, channel proto.UUID) (*Persona, error) {
+	if p, ok := a.pmap[channel]; ok {
+		return p, nil
+	} else {
+		p, err := FindPersona(dbif, a.Id, channel)
```

Comment:
__[gometalinter]__
* [gotype] invalid operation: a (variable of type *Account) has no field or method Id